src/frontEnd/vueChecks/utils.js

```diff
@@ -0,0 +1,35 @@
+const { find, findSync } = require('find-in-files');
+const { fetchJSON } = require('../../utils');
+
+/*
+  ⚠️ scoped-style validation gonna be ommited when open tag is multi-line. For example:
+    <style
+      lang="scss"
+      scoped
+    >
+      ...
+    </style>
+*/
+
+module.exports.checkScopedFiles = async testPath => {
+  const results = await findSync('<style', `${testPath}/src`, '.vue$');
+  return Object.values(results).every(({ line }) => line[0].includes('scoped'));
+};
+
+module.exports.checkVueTemplate = async testPath => {
+  const badFiles = await findSync(/[Vv]ue.component\(/, `${testPath}/src`, '.js$');
+  return !Object.keys(badFiles).length;
+};
+
+module.exports.checkVuexUse = async testPath => {
+  const response = await find(/^import [Vv]uex from 'vuex'/, `${testPath}/src/store`, 'index.js$')
+    .then(results => console.log('Vuex results', results) || !!Object.keys(results).length)
+    .catch(() => false);
```

Comment:
I think this is unnecesary

Reply:
I put that for the case that `src/store` doesn't exists.